lib/_stream_wrap.js

```diff
@@ -10,9 +10,11 @@ function StreamWrap(stream) {
 
   this.stream = stream;
 
+  this.queue = null;
```

Comment:
@indutny I am trying to understand this better. Theoretically, there will be no lookup in a queue, the first-in will be dequeued, right? If we are going to arbitrarily remove, then why not a `Map`?